types/event_bus.go

```diff
@@ -52,6 +52,10 @@ func (b *EventBus) OnStop() {
 	b.pubsub.OnStop()
 }
 
+func (b *EventBus) OnReset() error {
+	return b.pubsub.OnStart()
```

Comment:
why not `OnReset`?

Reply:
`pubsub.Sever` does not implement `OnReset()` and `OnStart()` starts the server as intended, do you want me to add the missing method in `tendermint/tmlibs` ? And then should I point glide.yaml to the master branch of tmlibs ?

Reply:
> do you want me to add the missing method in tendermint/tmlibs ?

yes, please

```go
func (s *Server) OnReset() error {	return nil }
```

> And then should I point glide.yaml to the master branch of tmlibs ?

no, point it to commit in tmlibs

Reply:
Want me to do it?

Reply:
It's not an urgent matter, i can take care of it when I'll have a few minutes, but thank's for asking :)